Sources/Architecture/X86/HardwareBreakpointManager.cpp

```diff
@@ -71,6 +72,12 @@ ErrorCode HardwareBreakpointManager::enableLocation(Site const &site) {
     return error;
   }
 
+  error = thread->writeDebugReg(kStatusRegIdx, 0);
```

Comment:
I think I understand why this is required, but wouldn't it make more sense to put it in `enable()` or someplace similar? The current code will clear the status register for every single location we enable.